src/sqlfluff/core/rules/std/L044.py

```diff
@@ -0,0 +1,83 @@
+"""Implementation of Rule L044."""
+
+from sqlfluff.core.rules.base import BaseCrawler, LintFix, LintResult
+from sqlfluff.core.rules.doc_decorators import (
+    document_configuration,
+    document_fix_compatible,
+)
+
+
+@document_configuration
+@document_fix_compatible
+class Rule_L044(BaseCrawler):
+    """Use consistent syntax to express "count number of rows".
+
+    COUNT(*) and COUNT(1) are equivalent syntaxes in many SQL engines
+    due to optimizers interpreting these instructions as
+    "count number of rows in result".
+
+    The ANSI-92_ spec mentions the COUNT(*) syntax specifically as
+    having a special meaning:
+
+        If COUNT(*) is specified, then
+        the result is the cardinality of T.
+
+    So by default, SQLFluff enforces the consistent use of COUNT(*).
+
+    If the SQL engine you work with, or your team, prefers COUNT(1)
+    over COUNT(*) you can configure this rule to consistently enforce COUNT(1).
+
+    .. _ANSI-92: http://msdn.microsoft.com/en-us/library/ms175997.aspx
+
+    | **Anti-pattern**
+
+    .. code-block:: sql
+
+        select
+            count(1)
+        from table_a
+
+    | **Best practice**
+    |   Use count(*) unless specified otherwise by config ``prefer_count_1``
+
+    .. code-block:: sql
+
+        select
+            count(*)
+        from table_a
+
+    """
+
+    config_keywords = ["prefer_count_1"]
+
+    def _eval(self, segment, **kwargs):
+        """Find rule violations and provide fixes."""
+        if segment.is_type("function"):
+
+            if self.prefer_count_1 and segment.raw_upper == "COUNT(*)":
```

Comment:
Will this work as intended if there are spaces, e.g. `COUNT( * )`?

Reply:
Now it should handle counts that have whitespaces, thanks for catching that